wpbm/info.json

```diff
@@ -36,8 +36,7 @@
       "defaultLanguage": "en-US"
     }
   },
-  "version_code": "5",
   "embed_zim": false,
+  "zim_url": "https://download.kiwix.org/zim/wikipedia/wikipedia_bm_all_maxi_2019-10.zim"
   "enforced_lang": "bm",
```

Comment:
I don't think we will be releasing this custom app but that `:` does need to be there